src/LclDckr/DockerClient.cs

```diff
@@ -312,11 +312,21 @@ public ICollection<ContainerInfo> Ps(bool all = false, IEnumerable<IFilter> filt
 
                     process.WaitForExit((int) TimeSpan.FromSeconds(10).TotalMilliseconds);
 
-                    if (process.ExitCode != 0 && breakOnError)
-                        throw new TimeoutException($"An error has occured. {process.StandardError.ReadToEnd()}");
-
-                    if (process.StandardOutput.ReadToEnd().Contains(desiredLog))
-                        return;
+                    if (!process.HasExited)
+                    {
+                        var output = process.StandardOutput.ReadToEnd();
+
+                        if (output.Contains(desiredLog))
```

Comment:
What if the process has not existing and the output does not contain the desired log yet?

Reply:
Argh, hidden by the diff view.